subprojects/core/src/main/java/org/gradle/api/internal/tasks/CacheableTaskOutputCompositeFilePropertyElementSpec.java

```diff
@@ -59,8 +59,8 @@ public OutputType getOutputType() {
     }
 
     @Override
-    public SnapshotNormalizationStrategy getSnapshotNormalizationStrategy() {
-        return parentProperty.getSnapshotNormalizationStrategy();
+    public PathNormalizationStrategy getPathNormalizationStrategy() {
+        return parentProperty.getPathNormalizationStrategy();
     }
```

Comment:
Not necessary, we can return the output strategy here. We should probably extract this logic to some common abstract type for all output properties.